lib/src/native/rtc_video_view_impl.dart

```diff
@@ -13,13 +13,16 @@ class RTCVideoView extends StatelessWidget {
     Key key,
     this.objectFit = RTCVideoViewObjectFit.RTCVideoViewObjectFitContain,
     this.mirror = false,
+    this.filterQuality = FilterQuality.low
```

Comment:
Let's add `,` at the end, to reduce diff when any additional parameter will be added.